docs/static/settings/monitoring-settings.asciidoc

```diff
@@ -38,6 +38,10 @@ If your {es} is protected with basic authentication, these settings provide the
 username and password that the Logstash instance uses to authenticate for
 shipping monitoring data.
 
+`xpack.monitoring.elasticsearch.proxy`::
```

Comment:
Remove `xpack.` prefix

Reply:
All the other settings are still prefixed with `xpack.` in the 7.x branch. Do you want I remove all of them ?

Reply:
No, in `7.x` we dropped the prefix `xpack.` for the `monitoring` part, https://github.com/elastic/logstash/blob/7.x/docs/static/settings/monitoring-settings.asciidoc

Reply:
Oh yes, my bad. I pushed the fix.